ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/ImageCve/ImageCvePage.tsx

```diff
@@ -296,6 +298,21 @@ function ImageCvePage() {
         tableLoading = deploymentDataRequest.loading;
     }
 
+    function trackEntityTabView(entityTab: EntityTab) {
+        analyticsTrack({
+            event: WORKLOAD_CVE_ENTITY_CONTEXT_VIEWED,
+            properties: {
+                type: entityTab,
+                page: 'CVE Detail',
```

Comment:
If CVE becomes superseded by vulnerability in UI this will be one more occurrence in the invisible iceberg.

The sooner to get long-term direction on terminology the better.